src/cli/generator/generator.rs

```diff
@@ -186,11 +186,16 @@ impl Generator {
                 None
             };
 
-            let mut llm_gen = InferTypeName::new(key);
+            let mut llm_gen = InferTypeName::new(key.clone());
```

Comment:
we should make `fieldName` optional in this case and if not specified by the user use LLM to infer the fieldName.